frontend/components/modal/ModalCommandPalette.vue

```diff
@@ -21,176 +26,113 @@
+      </DialogTitle>
+      <CommandPaletteItem @itemClicked="handleCloseModal" itemType="home" />
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="upcoming-events"
+      />
+      <!-- don't have notifications yet -->
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="notifications"
+      />
+      <!-- don't have discussions yet -->
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="discussions"
+      />
+
+      <!-- ORGANIZATIONS -->
+      <DialogTitle class="mt-3 flex justify-between font-display">
+        <p class="md:responsive-h3 pb-3 text-3xl font-bold">
+          {{ $t("components._global.organizations") }}
+        </p>
+      </DialogTitle>
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="organizations"
+      />
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="organizations"
+      />
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="organizations"
+      />
+
+      <!-- RESOURCES -->
+      <!-- TODO: enable this when we have a better idea of what resources -->
+      <!-- <DialogTitle class="mt-3 flex justify-between font-display">
+        <p class="md:responsive-h3 pb-3 text-3xl font-bold">
+          {{ $t("_global.resources") }}
+        </p>
+      </DialogTitle>
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="resources"
+      />
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="resources"
+      />
+      <CommandPaletteItem
+        @itemClicked="handleCloseModal"
+        itemType="resources"
+      /> -->
+
+      <!-- EVENTS -->
+      <DialogTitle class="mt-3 flex justify-between font-display">
+        <p class="md:responsive-h3 pb-3 text-3xl font-bold">
+          {{ $t("_global.events") }}
+        </p>
+      </DialogTitle>
+      <CommandPaletteItem @itemClicked="handleCloseModal" itemType="events" />
+      <CommandPaletteItem @itemClicked="handleCloseModal" itemType="events" />
+      <CommandPaletteItem @itemClicked="handleCloseModal" itemType="events" />
+    </div>
   </ModalBase>
 </template>
 
 <script setup lang="ts">
 import { IconMap } from "~/types/icon-map";
+import { DialogTitle } from "@headlessui/vue";
 
+// TODO: maybe get rid of isOpen, modalIsOpen and modalShouldClose, since all we really
+//  need is the 'closeModal' emit in 'handleCloseModal'
 const props = defineProps<{
   isOpen: boolean;
 }>();
-
-const localePath = useLocalePath();
-
 const modalIsOpen = computed(() => props.isOpen);
 const modalShouldClose = ref(false);
 
+const searchTerm = ref("");
+// const searchResults = []
+
 const emit = defineEmits(["closeModal"]);
 const handleCloseModal = () => {
-  modalShouldClose.value = true;
   emit("closeModal");
-  modalShouldClose.value = false;
 };
+
+// Watch the searchTerm ref variable
```

Comment:
Personal nitpick for me is periods at the end of own-line comments :) I normally do capitalization and periods at the end if it's its own line, and no capitalization and no period if it's inline 😊